servicelayer/archive/gs.py

```diff
@@ -121,3 +132,45 @@ def generate_url(self, content_hash, file_name=None, mime_type=None):
         return blob.generate_signed_url(expire,
                                         response_type=mime_type,
                                         response_disposition=disposition)
+
+    def store_export(self, namespace, file_path, mime_type=None):
+        file_path = ensure_posix_path(file_path)
+        file_name = safe_filename(file_path, default='data')
+        store_path = '{0}/{1}'.format(namespace, file_name)
+
+        for attempt in service_retries():
+            try:
+                blob = Blob(store_path, self.bucket)
+                blob.upload_from_filename(file_path, content_type=mime_type)
+            except FAILURES:
+                log.exception("Store error in GS")
+                backoff(failures=attempt)
+
+    def load_export(self, namespace, file_name, temp_path=None):
```

Comment:
Do we need `load_export`?

Reply:
Right, we don't. I assumed we need it for serving the exports in a dashboard but we can directly use the URL except in case of file system backend.